lib/yocaml_cmark/yocaml_cmark.mli

```diff
@@ -0,0 +1,22 @@
+(** Describing documents using a Markup language is very common in the
+    {i Blogosphere} (rather than describing all the formatting of document
+    content in HTML, using <p>, <strong> and co).
+
+    [Common Mark] is a very popular markup language (did you get the joke,
+    up/down) and, fortunately, OCaml has several one very good libraries for
+    turning Common Mark into HTML. This library is a wrapper around
+    {{:https://github.com/dbuenzli/cmarkit} cmarkit}, an excellent Common Mark
+    conversion library. *)
```

Comment:
It is a This is a bit of nitpicking but would it be possible to edit the comment :D to make it more relevant (no need to find a lame joke).